client/container_list.go

```diff
@@ -2,9 +2,11 @@ package client
 
 import (
 	"encoding/json"
+	"errors"
 	"net/url"
 	"strconv"
 
+	"github.com/docker/docker/pkg/version"
```

Comment:
@calavera regarding the version package, what is your opinion about creating the same one inside of `engine-api` to get rid of the dependency on `docker/docker`? It's a relatively small package, and I feel it's unnecessary to add external dependency just for that.